draft-ietf-tls-tls13.md

```diff
@@ -784,16 +782,16 @@ the following bytes:
 #  The TLS Record Protocol
 
 The TLS Record Protocol is a layered protocol. At each layer, messages
-may include fields for length, description, and content. The Record
+may include fields for length, description, and content.  The TLS Record
 Protocol takes messages to be transmitted, fragments the data into
 manageable blocks, protects the records, and transmits the
 result. Received data is decrypted and verified, reassembled, and then
 delivered to higher-level clients.
 
-Four protocols that use the record protocol are described in this document: the
-handshake protocol, the alert protocol, and
+Four protocols that use the TLS Record Protocol are described in this document: the TLS
+Handshake Protocol, the Alert Protocol, and
```

Comment:
The fourth was change_cipher_spec. That's gone now, so yeah, it needs updating to three.